application/core/LSYii_Application.php

```diff
@@ -711,4 +714,28 @@ public function getPublicBaseUrl($absolute = false)
         }
         return $baseUrl;
     }
+
+    /**
+     * get the current id of connected user,
+     * check if user exist before return for security
+     * @return int|null user id
+     */
+    public function getCurrentUserId()
+    {
+        if (App() instanceof CConsoleApplication) {
```

Comment:
Maybe remove it ? Just leave `if(empty(App()->session['loginID'])) {`

Reply:
Yes, seems better. 

Also, I am not sure CConsoleApplication will create LSYii_Application